web-ui/src/components/reviews/periods/ReviewPeriodStepper.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import optionsArr from './reviewStatus.json';
+import { toInteger } from 'lodash';
+
+export default function ReviewPeriodStepper({ reviewPeriod }) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  useEffect(() => {
+    let index = 0;
```

Comment:
I think there's a shorter way to do this. Maybe something like this:

```js
const index = optionsArr.findIndex(opt => opt.option === revPeriod);
```

Reply:
This worked fine, thanks for the suggestion.